### DISCUSSION TITLE
Smooth scrolling with UIScrollView is now in main.

### DISCUSSION THREAD

**Discussion by migueldeicaza** [migueldeicaza/SwiftTerm#260] [General]
I have finally merged the UIScrollView-based support in SwiftTerm for iOS, which provides a buttery smooth scrolling behavior.  

In addition, I have been doing a number of improvements to the selection on the iOS view and among others it should behave closer to other native controls on iOS.

**Comment by austincondiff** [comment]
Is this possible for macOS?

**Reply by migueldeicaza** [reply]
It might, but it feels that the current behavior matches best each native platform expectations.

I am open to it, just don't have any urgency to do it.
